src/components/molecules/NavBar/playa.scss

```diff
@@ -79,6 +79,7 @@ $sand: #937c63;
         height: 40px;
         width: 40px;
         background-color: rgba($black, 0.4);
+        border: none;
```

Comment:
Why removing border here?

Reply:
Using `none` and `inherit` isn't very popular/best practice and there is a reason for it. Compare staging to your branch and you will see the differences.

Just some examples:

This branch poll:
![branch-poll](https://user-images.githubusercontent.com/18435853/125596193-aef5ee80-c809-4e67-b12f-938b2eb37552.png)

Staging poll:
![staging-poll](https://user-images.githubusercontent.com/18435853/125596243-327d654b-12c8-45b3-b87a-20b0963cbe1d.jpg)


This branch schedule:
![branch-schedule](https://user-images.githubusercontent.com/18435853/125596272-8b2d8adc-8f61-4a0c-9fe3-bf89ca65983c.jpg)


Staging schedule:
![staging-schedule](https://user-images.githubusercontent.com/18435853/125596297-a46f933b-a4a9-44ab-9ba1-0c2ab003de8d.jpg)


I'm pretty sure these visual issues are coming from all of the inheritance and `none` values. This should be investigated and fixed before merging.

These 2 are the quick ones that I noticed, I saw that you change MapRoom's styles too, it might be visually affected as well.

Reply:
@denisdimitrov  I got the same issue after pulling the latest 'staging'. Continue work on that.